<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>alg1</class>
 <widget class="QWidget" name="alg1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTextBrowser" name="alg">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>520</width>
     <height>420</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Serif</family>
     <pointsize>20</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0);
font: 20pt &quot;MS Serif&quot;;

</string>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Serif'; font-size:20pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; color:#69ffb4;&quot;&gt;Выберите форму чисел&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="alg_text">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>240</y>
     <width>520</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 10pt &quot;Century Gothic&quot;;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="oper">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>100</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(255,136,121);
border: 1px solid rgb(40, 91, 174);
font: 8pt &quot;Century Gothic&quot;;

</string>
   </property>
   <item>
    <property name="text">
     <string>Операции</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>+</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>-</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>*</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>/</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="podtv">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>183</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
background-color:rgb(255,136,121);
border: 1px solid rgb(40, 91, 174);;
border-radius:8px;
font: 8pt &quot;Century Gothic&quot;;
}
QPushButton:hover:!pressed
{
background-color:rgb(255,164,141);
}

</string>
   </property>
   <property name="text">
    <string>Подтвердить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="rez">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>183</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>8</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
background-color:rgb(255,136,121);
border: 1px solid rgb(40, 91, 174);;
border-radius:8px;
font: 8pt &quot;Century Gothic&quot;;
}
QPushButton:hover:!pressed
{
background-color:rgb(255,164,141);
}

</string>
   </property>
   <property name="text">
    <string>Рассчитать</string>
   </property>
  </widget>
  <widget class="QLabel" name="atan">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>287</y>
     <width>520</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 10pt &quot;Century Gothic&quot;;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="AAA1">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>330</y>
     <width>520</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 10pt &quot;Century Gothic&quot;;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="ramka">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>500</y>
     <width>520</width>
     <height>160</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fixedsys</family>
     <pointsize>8</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(255,136,121);
border: 1px solid rgb(40, 91, 174);
border-radius:20px;
font: 8pt &quot;Fixedsys&quot;;</string>
   </property>
   <property name="plainText">
    <string>                   Ответ в Алгебраической форме


                 Ответ в Тригонометрической форме


                  Ответ в Экспоненциальноой форме</string>
   </property>
  </widget>
  <widget class="QComboBox" name="first">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>148</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(255,136,121);
border: 1px solid rgb(40, 91, 174);
font: 8pt &quot;Century Gothic&quot;;

</string>
   </property>
   <item>
    <property name="text">
     <string>Форма первого числа</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Алгебраическая</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Тригонометрическая</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Экспоненциальная</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="second">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>218</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(255,136,121);
border: 1px solid rgb(40, 91, 174);
font: 8pt &quot;Century Gothic&quot;;

</string>
   </property>
   <item>
    <property name="text">
     <string>Форма второго числа</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Алгебраическая</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Тригонометрическая</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Экспоненциальная</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="clear">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>380</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
background-color:rgb(255,136,121);
border: 1px solid rgb(40, 91, 174);;
border-radius:8px;
font: 8pt &quot;Century Gothic&quot;;
}
QPushButton:hover:!pressed
{
background-color:rgb(255,164,141);
}

</string>
   </property>
   <property name="text">
    <string>Очистить</string>
   </property>
  </widget>
  <widget class="QLabel" name="back">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>520</width>
     <height>420</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="real1">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>-100</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
QLineEdit
{
background-color:rgb(255,136,121);
border: 1px solid rgb(40, 91, 174);
border-radius:8px;
font: 75 12pt &quot;System&quot;;
}
QLineEdit:hover:!pressed
{
background-color:rgb(255,164,141);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="imag1">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>-200</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
QLineEdit
{
background-color:rgb(255,136,121);
border: 1px solid rgb(40, 91, 174);
border-radius:8px;
font: 75 12pt &quot;System&quot;;
}
QLineEdit:hover:!pressed
{
background-color:rgb(255,164,141);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="imag2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>500</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
QLineEdit
{
background-color:rgb(255,136,121);
border: 1px solid rgb(40, 91, 174);
border-radius:8px;
font: 75 12pt &quot;System&quot;;
}
QLineEdit:hover:!pressed
{
background-color:rgb(255,164,141);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="real2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>500</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
QLineEdit
{
background-color:rgb(255,136,121);
border: 1px solid rgb(40, 91, 174);
border-radius:8px;
font: 75 12pt &quot;System&quot;;
}
QLineEdit:hover:!pressed
{
background-color:rgb(255,164,141);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="imag2_push">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>140</y>
     <width>160</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0, 189, 132,0);
border: 1px solid rgba(40, 91, 174,0);
border-radius:8px;
font: 75 12pt &quot;System&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="imag1_push">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>60</y>
     <width>160</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0, 189, 132,0);
border: 1px solid rgba(40, 91, 174,0);
border-radius:8px;
font: 75 12pt &quot;System&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="real1_push">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>60</y>
     <width>160</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0, 189, 132,0);
border: 1px solid rgba(40, 91, 174,0);
border-radius:8px;
font: 75 12pt &quot;System&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="real2_push">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>110</y>
     <width>160</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0, 189, 132,0);
border: 1px solid rgba(40, 91, 174,0);
border-radius:8px;
font: 75 12pt &quot;System&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTextBrowser" name="alg_2">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>520</width>
     <height>420</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Serif</family>
     <pointsize>20</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0);
font: 20pt &quot;MS Serif&quot;;

</string>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Serif'; font-size:20pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; color:#69ffb4;&quot;&gt;Введите числа и выберите знак&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <zorder>back</zorder>
  <zorder>alg_2</zorder>
  <zorder>alg</zorder>
  <zorder>ramka</zorder>
  <zorder>alg_text</zorder>
  <zorder>oper</zorder>
  <zorder>atan</zorder>
  <zorder>AAA1</zorder>
  <zorder>rez</zorder>
  <zorder>first</zorder>
  <zorder>second</zorder>
  <zorder>clear</zorder>
  <zorder>podtv</zorder>
  <zorder>real1</zorder>
  <zorder>imag1</zorder>
  <zorder>imag2</zorder>
  <zorder>real2</zorder>
  <zorder>imag2_push</zorder>
  <zorder>imag1_push</zorder>
  <zorder>real1_push</zorder>
  <zorder>real2_push</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>clear</sender>
   <signal>clicked()</signal>
   <receiver>AAA1</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>435</x>
     <y>399</y>
    </hint>
    <hint type="destinationlabel">
     <x>426</x>
     <y>367</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clear</sender>
   <signal>clicked()</signal>
   <receiver>atan</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>450</x>
     <y>402</y>
    </hint>
    <hint type="destinationlabel">
     <x>429</x>
     <y>312</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clear</sender>
   <signal>clicked()</signal>
   <receiver>alg_text</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>462</x>
     <y>396</y>
    </hint>
    <hint type="destinationlabel">
     <x>429</x>
     <y>236</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clear</sender>
   <signal>clicked()</signal>
   <receiver>imag1</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>409</x>
     <y>396</y>
    </hint>
    <hint type="destinationlabel">
     <x>407</x>
     <y>-87</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clear</sender>
   <signal>clicked()</signal>
   <receiver>imag2</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>453</x>
     <y>394</y>
    </hint>
    <hint type="destinationlabel">
     <x>415</x>
     <y>508</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clear</sender>
   <signal>clicked()</signal>
   <receiver>real1</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>457</x>
     <y>393</y>
    </hint>
    <hint type="destinationlabel">
     <x>223</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clear</sender>
   <signal>clicked()</signal>
   <receiver>real2</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>474</x>
     <y>400</y>
    </hint>
    <hint type="destinationlabel">
     <x>275</x>
     <y>507</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
